fl4health/clients/instance_level_privacy_client.py

```diff
@@ -47,16 +45,17 @@ def setup_client(self, config: Config) -> None:
         super().setup_client(config)
 
         # Configure DP training
-        self.setup_opacus_objects()
+        self.setup_opacus_objects(config)
 
-    def setup_opacus_objects(self) -> None:
+    def setup_opacus_objects(self, config: Config) -> None:
```

Comment:
Since we are already having to do this interval validation on the client, should we also move the validation internally on the server side. That way to the user doesn't need to know anything about the validation function you wrote and its a more seamless experience. I may be overlooking some details though so let me know

Reply:
The only reason I didn't do that is because then we'd have to pass the models to the strategies for manipulation. This is sort of already available in SCAFFOLD strategies, but it isn't for BasicFedAvg, which is used by this client. If we did that, I'd want to create my own strategies, so that it isn't hardcoded into SCAFFOLD or BasicFedAvg. That's because Opacus doesn't play nice with some of the more sophisticated models (like Fed-IXI's U-net).

Reply:
Maybe we can put this in the backlog to consider?

Reply:
sounds good!

Reply:
Added to the backlog.